src/TinyMCECombinedGenerator.php

```diff
@@ -0,0 +1,289 @@
+<?php
+
+namespace SilverStripe\TinyMCE;
```

Comment:
```suggestion
namespace SilverStripe\HtmlEditorTinyMCE;
```
So that it better matches the name of the repo?

Reply:
I'd prefer to leave it as is because:
1. It's easier to remember
2. I've already done it with the current name, and changing it is subjective and therefore work for little if any benefit
3. There's no need for a namespace to closely match the package name, we have no standards around this, and if I spent a second to look I could probably show you a bunch of cases where we don't currently do that

> Right now it reads like a Silverstripe fork of TinyMCE, which it's not.

TinyMCE is JavaScript so that doesn't really make sense

Reply:
Ok keep as `\TinyMCE`